nextflow.config

```diff
@@ -51,7 +51,8 @@ profiles {
       process.executor = 'google-lifesciences'
       process.container = 'nextflow/rnaseq-nf:latest'
       workDir = 'gs://rnaseq-nf/scratch' // <- replace with your own bucket!
-      google.region  = 'europe-west2'
+      google.location  = 'europe-west2'
+      google.region = 'europe-west1'
   }  
```

Comment:
I'm still not sure to get what's the benefits of specifying both the `region` and the `location`. Since NF [defaults the location to the same setting as for the region](https://github.com/nextflow-io/nextflow/blob/48e5e4a889b8a168eecb09c83dc648651d02d16c/modules/nf-google/src/main/nextflow/cloud/google/lifesciences/GoogleLifeSciencesConfig.groovy#L114) (or the zone), would that not be enough (at least for a minimal example)?

Reply:
Currently, the only [supported locations for Cloud Life Sciences API](https://cloud.google.com/life-sciences/docs/concepts/locations) are a subset of the overall GCP regions (currently: us-central1 (Iowa) & europe-west2 (London)), whereas supported regions are everywhere Compute Engine is (pretty much everywhere). So as location rollout continues, there is a gap between them and possible for users to put a region that is not a currently supported location, which will throw an error.

Reply:
Got it now, basically `location` identifies the API endpoint, instead region/zone where the computation is allocated. 

However, I really prefer to leave as optional settings redundant or that can be inferred. What if I implement a smarter fallback for the location so that it tries to use `us-central1` or europe-west2` depending on the region/zone and warn the users if a not supported one is specified. Then, we'll update it as new locations are added.

Reply:
That sounds like a better idea. For a solution that will 'auto-update' you can get the list of supported regions with a call to cloudLifeSciencesService.projects().locations().list() There are [API details here](https://cloud.google.com/life-sciences/docs/reference/rest/v2beta/projects.locations/list), including an example using the Java client library.

Reply:
Awesome! I like this!